src/themes/index.ts

```diff
@@ -1,7 +1,7 @@
-import { theme } from './converted'
+import { theme, lightTheme } from './converted'
 
-export async function loadTheme(editor: typeof import('monaco-editor-core').editor) {
+export async function loadTheme(editor: typeof import('monaco-editor-core').editor, dark = true) {
     const themeName = 'vs-code-theme-converted';
-    editor.defineTheme(themeName, theme);
+    editor.defineTheme(themeName, dark ? theme : lightTheme);
```

Comment:
也许这里可以同时定义 dark 和 light 主题

```suggestion
editor.defineTheme('vs-code-theme-converted-dark', theme);
editor.defineTheme('vs-code-theme-converted-light', lightTheme);
```

然后我们可以用 monaco.editor.setTheme('vs-code-theme-converted-light') 来切换主题